pwndbg/commands/peda.py

```diff
@@ -19,15 +19,21 @@ def getfile() -> None:
     print(repr(pwndbg.auxv.get().AT_EXECFN))
 
 
-parser = argparse.ArgumentParser(description="Continue execution until an address or function.")
-parser.add_argument("target", type=str, help="Address or function to stop execution at")
+parser = argparse.ArgumentParser(
+    description="Continue execution until an address or function."
+)
+parser.add_argument("target", type=int, help="Address or function to stop execution at")
 
 
 @pwndbg.commands.ArgparsedCommand(parser, category=CommandCategory.NEXT)
 def xuntil(target) -> None:
+    running = pwndbg.gdblib.proc.alive
+    if not running:
+        print(message.notice("Process is not running. Starting process..."))
+        gdb.execute("starti", from_tty=False, to_string=True)
```

Comment:
Ideally, we want debugger agnostic way to run this if possible

Reply:
But don't worry about it for now since its not there yet; from discord:
![Uploading image.png…]()